src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_consent_manager_impl.h

```diff
@@ -46,7 +46,7 @@ namespace rc_rpc_plugin {
 class RCConsentManagerImpl : public RCConsentManager {
  public:
   RCConsentManagerImpl(
-      resumption::LastState& last_state,
+      resumption::LastStateWrapperPtr last_state,
```

Comment:
re-add and deprecate old signature

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508